### ISSUE TITLE
[enhancement] Print better error in the case of ownership issues

### ISSUE THREAD

**Issue by chiraag-nataraj** [netblue30/firejail#2743]
If I use `private-bin` to whitelist a binary in `/usr/bin` which happens to have the wrong owner (for example, my user), `firejail` prints a very opaque and unhelpful error. The context this came up in is `protonmail-bridge`, which (for some godforsaken reason) installed with _my user_ as the owner (going to report that as a bug to them). This meant that `private-bin` failed (as it should), but the error message was unhelpful in figuring out what the problem was.

**Comment by rusty-snake** [comment]
If we want to print better error messages, we can also fix this:
```
$ firejail --name  # Invalid because it need =NAME
Error: invalid --name command line option
$ firejail --doesnt_exists # Invalid because it doesn't exists
Error: invalid --doesnt_exists command line option
```

**Comment by rusty-snake** [comment]
or this (`--quiet` and `--debug`):
```
$ firejail --quiet --debug XXX # or firejail --debug --quiet
Error: invalid --debug command line option
```

**Comment by netblue30** [comment]
Closed for now, any other problem go to #4275
